<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificatorWingWidget</class>
 <widget class="QWidget" name="ModificatorWingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="anchorLayout">
     <item>
      <spacer name="horizontalSpacer_33">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAnchor">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Anchor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxAnchorX">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxAnchorY">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxAnchorZ">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-1000000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="symmetryLayout">
     <item>
      <spacer name="horizontalSpacer_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSymmetry">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Symmetry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSymmetry"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sweepLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSweep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sweep</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxSweep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-89.000000000000000</double>
       </property>
       <property name="maximum">
        <double>89.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelChord">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Chord %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxSweepChord">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.250000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dihedralLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDihedral">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Dihedral</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxDihedral">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-89.000000000000000</double>
       </property>
       <property name="maximum">
        <double>89.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDihedralChord">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Chord %</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxDihedralChord">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.250000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="areaLayout">
     <item>
      <spacer name="horizontalSpacer_32">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelArea">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reference area, the area of the projected wing on the plane formed by the wing major direction and the wing deep direction. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Top Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxIsAreaConstant">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Const</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxArea">
       <property name="keyboardTracking">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999999999999998278261272554585856747747644714015897553975120217811154108416.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="spanLayout">
     <item>
      <spacer name="horizontalSpacer_31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSpan">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Span</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxIsSpanConstant">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Const</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxSpan">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>99999999999999998278261272554585856747747644714015897553975120217811154108416.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="aRLayout">
     <item>
      <spacer name="horizontalSpacer_34">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>26</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAR">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>AR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxIsARConstant">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Const</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBoxAR">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.010000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000013287555072.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
